Packages/com.unity.inputsystem/Documentation~/PlayerInput.md

```diff
@@ -12,14 +12,12 @@ The **Player Input** component represents a single player, and that player's ass
 
 ![PlayerInput](Images/PlayerInput.png)
 
-*Above, the Player Input component as visible in the inspector, with an Actions Asset called "ExampleActions" assigned.*
+*Above, the Player Input component as visible in the inspector.*
 
-Each [`PlayerInput`](../api/UnityEngine.InputSystem.PlayerInput.html) instance represents a separate player or user. You can use multiple `PlayerInput` instances at the same time (although not on the same `GameObject`) to represent local multiplayer setups. The Input System pairs each player to a unique set of Devices that the player uses exclusively, but you can also manually pair Devices in a way that enables two or more players to share a Device (for example, left/right keyboard splits or hot seat use).
+Each [`PlayerInput`](../api/UnityEngine.InputSystem.PlayerInput.html) instance represents a separate player or user. You can use multiple `PlayerInput` instances at the same time (although not on the same `GameObject`) to implement local multiplayer setups. The Input System pairs each player to a unique set of Devices that the player uses exclusively, but you can also manually pair Devices in a way that enables two or more players to share a Device (for example, left/right keyboard splits or hot seat use).
```

Comment:
I think you need a PlayerManager Component to do the paring of each player to a unique set of devices. So I think this is wrong when using the PlayerInput component on its own.

Reply:
You do not need a PlayerManager to achieve the pairing. I believe just need to have multiple PlayerInput component instances. PlayerManager is needed to get the "press to join behavior". Have you observed something different @lyndon-unity ?

Reply:
Ignore this comment as further testing confirmed device binding does work without player input manager.